.env.example

```diff
@@ -183,3 +186,17 @@ CSP_POLICY=
 EDGE_CONFIG=
 
 NEXT_PUBLIC_MINUTES_TO_BOOK=5 # Minutes
+
+# - ORGANIZATIONS *******************************************************************************************
+# Enable Organizations non-prod domain setup, works in combination with organizations feature flag
+# This is mainly needed locally, because for orgs to work a full domain name needs to point
+# to the app, i.e. app.cal.local instead of using localhost, which is very disruptive
+#
+# This variable should only be set to 1 or true if you are in a non-prod environment and you want to
+# use organizations
+ORGANIZATIONS_ENABLED=
```

Comment:
Maybe we can enable it automatically on dev by checking if NEXT_PUBLIC_WEBAPP_URL is a organisation supported domain or not(which is basically not localhost)

Reply:
Yeah, functionality wise the feature flag can be turned off. Not entirely sure what to do here.

Reply:
Marking it resolved. Let's leave it as is for now.